agent/xds/config.go

```diff
@@ -192,8 +193,15 @@ func ToOutlierDetection(p *structs.PassiveHealthCheck) *envoy_cluster_v3.Outlier
 	}
 
 	if p.EnforcingConsecutive5xx != nil {
+		// NOTE: EnforcingConsecutive5xx must be greater than 0 for ingress-gateway
 		od.EnforcingConsecutive_5Xx = &wrappers.UInt32Value{Value: *p.EnforcingConsecutive5xx}
```

Comment:
It looks like `wrappers` comes from the deprecated GH package. I don't think it's critical to this PR, but if we went through the process of upgrading the proto generation to the `google.golang` version, we might want to clean it out completely. Not sure how much of a lift this is, though. Not blocking.

Reply:
Good point, I can change it in another PR. I want to make sure it won't break the backport assist PRs with out of band changes.